serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -60,8 +60,6 @@ The previous command creates a Maven Quarkus project in the `serverless-workflow
 * `quarkus-resteasy-jackson`: Adds support for RESTEasy, which is required by the generated REST resources that are used to start the flow process using an HTTP request.
 * `quarkus-smallrye-openapi`: Adds support for Swagger documentation when you run the application in development mode.
 * `--no-code`: Prevents workflow example code from being generated.
-
-The SwaggerUI is available at `http://localhost:8080/q/swagger-ui/` when you run the application.
```

Comment:
I think this should be left here as it probably refers to the part about
```
* `quarkus-smallrye-openapi`: Adds support for Swagger documentation when you run the application in development mode.
```

Reply:
I moved it out of the `Quarkus CLI` box because it is information relevant to all the three options you can choose to create/run the project. Or is there maybe a better place to have it?

Reply:
Probably Running your workflow application is a better place? As Swagger UI can be used to execute requests there.

Reply:
Yeah, that makes sense. Thanks. I moved it there and linked it with the `quarkus-smallrye-openapi` description.